{
	"id": "oval:com.redhat.cve:def:202144568",
	"version": "636",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2021-44568 libsolv: heap-overflows in resolve_dependencies function (low)",
		"reference": [
			{
				"ref_id": "CVE-2021-44568",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2021-44568",
				"source": "CVE"
			}
		],
		"description": "DOCUMENTATION: A buffer over-read flaw was found in the test case reader in libsolv that created multiple out-of-bounds read symptoms. Depending on how client applications use libsolv, this flaw leads to a denial of service of the application if an attacker can supply crafted input to the test case reader. \n            STATEMENT: This flaw has been marked as Low impact because it is in the test case reader and is an out-of-bounds read.",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Low",
			"updated": {
				"date": "2022-10-15"
			},
			"cve": [
				{
					"text": "CVE-2021-44568",
					"cvss3": "6.3/CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:L/I:N/A:H",
					"cwe": "CWE-125",
					"href": "https://access.redhat.com/security/cve/CVE-2021-44568",
					"impact": "low",
					"public": "20220221"
				}
			],
			"affected": {
				"resolution": [
					{
						"state": "Affected",
						"component": [
							"libsolv",
							"python2-solv",
							"python3-solv"
						]
					}
				]
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:satellite:6.9",
					"cpe:/a:redhat:satellite:6.9::el7",
					"cpe:/a:redhat:satellite_capsule:6.9",
					"cpe:/a:redhat:satellite_capsule:6.9::el7"
				]
			},
			"issued": {}
		},
		"affected": {}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:201920387001",
										"comment": "python3-solv is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:201920387002",
										"comment": "python3-solv is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:201920387003",
										"comment": "libsolv is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:201920387004",
										"comment": "libsolv is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:201920387005",
										"comment": "python2-solv is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:201920387006",
										"comment": "python2-solv is signed with Red Hat redhatrelease2 key"
									}
								]
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:com.redhat.cve:tst:20134120011",
						"comment": "Red Hat Enterprise Linux 7 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.cve:tst:20134120012",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
